### DISCUSSION TITLE
No module named 'scattertext'

### DISCUSSION THREAD

**Discussion by gskarp** [JasonKessler/scattertext#104] [Q&A] [answered]
I have scattertext installed in my iMac as well in my MacBook. When trying to load it with 
`import scattertext as st`
I get a `ModuleNotFoundError: No module named 'scattertext'`

Since I am getting it in both computers, I wonder if there are any ideas how to overcome this error.

**Comment by JasonKessler** [comment, marked as answer]
It's tough to tell what's going on, but I'd recommend creating a new virtual environment with conda, activating it, running pip install scattertext, and then activating that virtual environment whenever you'd like to use the package.

Note that if you'd like to use jupyter, you'll have to go through some additional steps, along the lines of https://towardsdatascience.com/python-virtual-environments-jupyter-notebook-bb5820d11da8

**Reply by gskarp** [reply]
thanks a lot. I use jupyter, so probably that' s the reason.
